app/containers/UIKit/Section.tsx

```diff
@@ -30,8 +30,10 @@ const Accessory = ({ element, parser }: IAccessoryComponent) =>
 
 const Fields = ({ fields, parser, theme }: IFields) => (
 	<>
-		{fields.map(field => (
-			<Text style={[styles.text, styles.field, { color: themes[theme].bodyText }]}>{parser.text(field)}</Text>
+		{fields.map((field, index) => (
+			<Text key={index} style={[styles.text, styles.field, { color: themes[theme].bodyText }]}>
```

Comment:
https://react.dev/learn/rendering-lists#why-does-react-need-keys
See pitfall section

Reply:
This issue has not yet been resolved.

Reply:
Sir I find this one trivial as the field is of type `any` which means that using this as the key it can cause major bugs in future so that's the reason I have made use of the index here.

I would love to know suggestions.